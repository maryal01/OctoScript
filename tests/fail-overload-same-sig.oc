fn overload foo(list<int> l) -> int {
    return 0;
}

fn overload foo(list<int> l) -> bool {
    return true;
}
