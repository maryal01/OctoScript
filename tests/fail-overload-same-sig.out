Fatal error: exception Failure("Overloaded function foo with same signature declared twice")